tests/unit/client.py

```diff
@@ -766,3 +771,31 @@ def test_extension_list(self):
                     "hil.ext.switches.mock",
                     "hil.ext.switches.nexus",
                 ]
+
+
+class TestShowNetworkingAction:
+    """Test calls to show networking action method"""
+
+    def test_get_status(self):
+        """(successful) call to get_status"""
+        response = C.node.connect_network(
+                'node-01', 'eth0', 'net-01', 'vlan/native'
+                )
+        status_id = response['status_id']
+
+        response = C.node.show_networking_action(status_id)
+        assert response == {'status': 'PENDING',
+                            'node': 'node-01',
+                            'nic': 'eth0',
+                            'type': 'modify_port',
+                            'channel': 'vlan/native',
+                            'new_network': 'net-01'}
+
+        deferred.apply_networking()
+        response = C.node.show_networking_action(status_id)
+        assert response['status'] == 'DONE'
+
+    def test_get_status_fail(self):
+        """(unsuccessful) call to get_status"""
```

Comment:
s/get_status/show_networking_action/. There are a couple of other instances of this too.